packages/medusa/src/api/middlewares/invalidate-cached-events.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction, Request, Response } from "express"
+import { isDefined, MedusaError } from "medusa-core-utils"
+import { IEventBusService } from "../../interfaces/services/event-bus"
+
+export default () => {
```

Comment:
todo: No need to export a function that returns a function. We can directly export the middleware since there is no configuration required